src/library_pthread.js

```diff
@@ -5,7 +5,7 @@
 
 var LibraryPThread = {
   $PThread__postset: 'if (!ENVIRONMENT_IS_PTHREAD) PThread.initMainThreadBlock();',
-  $PThread__deps: ['$PROCINFO', '_register_pthread_ptr', 'emscripten_main_thread_process_queued_calls'],
+  $PThread__deps: ['$PROCINFO', '_register_pthread_ptr', 'emscripten_main_thread_process_queued_calls', '$ERRNO_CODES'],
```

Comment:
Was this missing also for asm.js?  How as that working?

Reply:
There are uses of this lower down, so this was always a bug, but it happens that something in how fastcomp sets things up did bring in this dependency by luck.